src/main/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/FagsakService.kt

```diff
@@ -24,6 +33,9 @@ class FagsakService(
     private val behandlingRepository: BehandlingRepository
 ) {
 
+    private val antallFagsakerOpprettetFraManuell =
```

Comment:
Ser at dette er kopiert fra ba-sak, men greier vi å finne på nytt navn?
Hva betyr egentlig antallFagsakerOpprettetFraManuell? Manuell hva?

Reply:
i ba-sak kan en fagsak opprettes både automatisk og manuellt. På sikt kommer vi også for ks til å opprette fagsaker på 2 måter, enten manuellt av saksbehandler eller automatisk når vi får en søknad fra søknadsdialogen